install_config/registry/accessing_registry.adoc

```diff
@@ -272,3 +272,99 @@ cf2616975b4a: Image successfully pushed
 Digest: sha256:3662dd821983bc4326bee12caec61367e7fb6f6a3ee547cbaff98f77403cab55
 ----
 ====
+
+[[access-reading-and-writing-image-signatures]]
+=== Reading and Writing Image Signatures
+The {product-title} integrated Docker Registry provides an "extensions" endpoint that
+allows you to write and read image signatures.
+The image signatures are stored in the {product-title} key-value store via Docker Registry API.
+
+[NOTE]
+====
+This endpoint is experimental and not supported by the upstream Docker Registry project.
+See link:https://docs.docker.com/registry/spec/api/[upstream API documentation] for
+general information about the Docker Registry API.
+====
+
+[[writing-image-signatures]]
+==== Writing Image Signatures
+In order to add a new signature to the Image, users can use the HTTP `PUT` method together
+with the JSON payload sent the extensions endpoint:
+
+====
+----
+PUT /extensions/v2/{namespace}/{name}/signatures/{digest}
+----
+====
+
+====
+----
+$ curl -X PUT --data @signature.json http://<user>:<token>@<registry-endpoint>:5000/extensions/v2/<namespace>/<name>/signatures/sha256:<digest>
```

Comment:
To be consistent, replace all the occurences of `sha256:<digest>` with just `<digest>`.